CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ a release.
 ---
 
 ## [Unreleased]
+###
+- Sortable: Fix duplicated positions when manually updating position on more than one object
```

Comment:
Please, rebase and place this item in the existing [`Fixed`](https://github.com/doctrine-extensions/DoctrineExtensions/blame/main/CHANGELOG.md#L22) section.

Reply:
@phansys done